tiny_dnn/activations/relu_layer.h

```diff
@@ -0,0 +1,36 @@
+/*
+    Copyright (c) 2017, Taiga Nomi, Karan Desai
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn {
+
+class relu_layer : public activation_layer {
+ public:
+  using activation_layer::activation_layer;
+
+  std::string layer_type() const override { return "relu-activation"; }
+
+  void forward_activation(const vec_t &x, vec_t &y) {
+    for (serial_size_t j = 0; j < x.size(); j++) {
+      y[j] = std::max(float_t(0), x[j]);
+    }
+  }
+
+  void backward_activation(const vec_t x,
```

Comment:
@karandesai-96 You should make 1st argument `x` to a reference `&` type. `const vec_t &x`